ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -567,3 +567,32 @@ def test_numpy_negative_binomial(
         p=p,
         size=size,
     )
+# laplace
+@helpers.handle_frontend_test(
+    fn_tree="numpy.random.laplace",
+    input_dtypes=helpers.get_dtypes("float", index=2),
+    mean=st.floats(allow_nan=False, allow_infinity=False, width=32),
+    scale=st.floats(allow_nan=False, allow_infinity=False, width=32, min_value=0),
+    size=st.tuples(st.integers(min_value=2, max_value=5), st.integers(min_value=2, max_value=5)),
+)
+def test_ivy_laplace(
```

Comment:
Since this is a frontend test, it should be `test_numpy_laplace`. You could refer to other test function names in the script if you have any confusions :slightly_smiling_face:

Reply:
Fixing both names and committing in another 15 mins. Thanks for the review @bipinKrishnan :D

Reply:
Fixed @bipinKrishnan ✅